driver/tests/src/test/scala/org/virtuslab/ideprobe/ProbeDriverTest.scala

```diff
@@ -250,12 +251,15 @@ final class ProbeDriverTest extends IntegrationTestSuite with Assertions {
     assertTrue(s"New Project dialog content: '$dialogContent' did not contain '$projectSdk'", dialogContent.contains(projectSdk))
   }
 
+  @tailrec
   private def retry[A](times: Int)(action: => A): A = {
     try {
       action
     } catch {
       case e: WaitForConditionTimeoutException =>
         if (times > 0) {
+          println("Failed to find element, retrying...")
```

Comment:
isn't the action generic? We might not always be looking for an element

Reply:
this exception will only be thrown from `find`. Anyway I just added it for debugging purposes as it failed again on this PR. Eventually, when we know what is wrong we can make this generic.